pkg/coscheduling/core/core.go

```diff
@@ -98,14 +98,14 @@ func NewPodGroupManager(pgClient pgclientset.Interface, snapshotSharedLister fra
 // 2. the total number of pods in the podgroup is less than the minimum number of pods
 // that is required to be scheduled.
 func (pgMgr *PodGroupManager) PreFilter(ctx context.Context, pod *corev1.Pod) error {
-	klog.V(5).Infof("Pre-filter %v", pod.Name)
+	klog.V(5).InfoS("Pre-filter", "pod", klog.KObj(pod))
 	pgFullName, pg := pgMgr.GetPodGroup(pod)
 	if pg == nil {
 		return nil
 	}
 	if _, ok := pgMgr.lastDeniedPG.Get(pgFullName); ok {
 		err := fmt.Errorf("pod with pgName: %v last failed in 3s, deny", pgFullName)
-		klog.V(6).Info(err)
+		klog.V(6).InfoS(err.Error(), "podGroup", pg)
```

Comment:
It seems the caller also dumps the error, so let's eliminate the logging here and just return the error:

```
return fmt.Errorf("pod with pgName: %v last failed in 3s, deny", pgFullName)
```